src/style.css

```diff
@@ -49,6 +49,7 @@
 .search-bar__input:focus {
   background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
   border-radius: 3px;
+  font-weight: bold;
```

Comment:
styles for font don't have to chage on focus, there styles for placeholder and styles for inputted text